collector/collector_test.go

```diff
@@ -27,6 +27,13 @@ func newTestCollector() *Collector {
 						{State: bk.String("scheduled"), AgentQueryRules: []string{"queue=default"}},
 					},
 				},
+				{
+					Pipeline: &bk.Pipeline{Name: bk.String("vicuñas")},
+					State:    bk.String("scheduled"),
+					Jobs: []*bk.Job{
+						{State: bk.String("scheduled"), AgentQueryRules: []string{"queue=default"}},
```

Comment:
I added this to the default queue because I couldn't find a way to add another; this means I had to change a bunch of other, unrelated numbers below. Is there a better way to do this?

Reply:
That test needs some cleaning up to split apart those concerns, but I'm fine with it for now.